sqlmesh/core/constants.py

```diff
@@ -40,15 +40,16 @@
 """The default directory for log files."""
 
 AUDITS = "audits"
+CACHE = ".cache"
+EXTERNAL_MODELS = "external_models"
 MACROS = "macros"
 METRICS = "metrics"
 MODELS = "models"
-EXTERNAL_MODELS = "external_models"
 SEEDS = "seeds"
 TESTS = "tests"
-CACHE = ".cache"
-SCHEMA_YAML = "schema.yaml"
 
+EXTERNAL_MODELS_YAML = "external_models.yaml"
+DEPRECATED_YAML = "schema.yaml"
```

Comment:
I liked the previous name better :D Now it's quite unclear which YAML has been deprecated. Sorry about the back and forth.